outputs.tf

```
@@ -1,3 +1,9 @@
+output "rollbar_project_name" {
+  value = data.rollbar_project.this.name
+
+  description = "The name of the Rollbar project."
+}
```

instead of returning a specific attribute, we usually return the whole resource/data source:

```suggestion
output "rollbar_project" {
  value = data.rollbar_project.this

  description = "The Rollbar project."
}
```